src/Controller/LookSlugController.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusShopTheLookPlugin\Controller;
+
+use Sylius\Component\Product\Generator\SlugGeneratorInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+final class LookSlugController extends Controller
```

Comment:
I think this should just be a controller action with extending anything

Reply:
Agree